resources/assets/lib/beatmap-discussions/editor-discussion-component.tsx

```diff
@@ -107,36 +277,42 @@ export default class EditorDiscussionComponent extends React.Component<Props> {
         suppressContentEditableWarning={true}
         {...this.props.attributes}
       >
-        <div className={osu.classWithModifiers(bn, classMods)}>
-          <div className={`${bn}__content`}>
+        <div className={osu.classWithModifiers(this.bn, classMods)}>
+          <div className={`${this.bn}__content`}>
             <div
-              className={`${bn}__selectors`}
+              className={`${this.bn}__selectors`}
               contentEditable={false} // workaround for slatejs 'Cannot resolve a Slate point from DOM point' nonsense
             >
               <EditorBeatmapSelector {...this.props} disabled={this.props.readOnly || !canEdit}/>
               <EditorIssueTypeSelector {...this.props} disabled={this.props.readOnly || !canEdit}/>
               <div
-                className={`${bn}__timestamp`}
+                className={`${this.bn}__timestamp`}
                 contentEditable={false} // workaround for slatejs 'Cannot resolve a Slate point from DOM point' nonsense
               >
                 <span title={canEdit ? timestampTooltip : ''}>
                   {this.props.element.timestamp || osu.trans('beatmap_discussions.timestamp_display.general')}
                 </span>
               </div>
               {unsavedIndicator}
+              <DraftsContext.Consumer>{this.nearbyIndicator}</DraftsContext.Consumer>
```

Comment:
I've mentioned this somewhere else but it looks like storing this to a variable and using it here and in the other place just like `unsavedIndicator` may work and save rendering same thing twice.